src/Sandbox.js

```diff
@@ -330,12 +367,12 @@ function packageInfoKey(env: Environment, packageInfo: PackageInfo) {
       },
     });
     if (process.env.ESY__TEST) {
-      packageInfo.__key = `${normalizedName}-${version || '0.0.0'}`;
+      packageInfo.__cachedPackageHash = `${normalizedName}-${version || '0.0.0'}`;
```

Comment:
So, why the need to test `if (process.env.ESY__TEST)` to determine the package hash. I thought it would already be taken into account.